src/library/scala/concurrent/Future.scala

```diff
@@ -101,9 +107,13 @@ trait Future[+T] extends Awaitable[T] {
    *  If the future has already been completed with a value,
    *  this will either be applied immediately or be scheduled asynchronously.
    *
+   *  Note that the returned value of `pf` will be discarded.
+   *
    *  $swallowsExceptions
    *  $multipleCallbacks
    *  $callbackInContext
+   *
+   * @group Callbacks
```

Comment:
Not sure there is a prio really. What does it pick by default? Lexicographical sort?